src/blocks/google-map/save.js

```diff
@@ -0,0 +1,35 @@
+import classnames from 'classnames';
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+
+const api_key = 'AIzaSyAsd_d46higiozY-zNqtr7zdA81Soswje4';
```

Comment:
Which API key is this? is this alright to be shipped in the plugin?

```suggestion
const apiKey = 'AIzaSyAsd_d46higiozY-zNqtr7zdA81Soswje4';
```

Reply:
Yes, It is free and doesn't harm any.

Still, I am discussing this with Vrunda and the team for the key restrictions. ( This has already been shipped in the previous version. )